src/hearings/containers/hearing-actuals/hearing-actuals-timing/hearing-actuals-timing.component.ts

```diff
@@ -0,0 +1,119 @@
+    const actualTime = hearingActuals.hearingActuals.actualHearingDays[0].hearingStartTime;
+    return actualTime ? actualTime.match(this.timeMatcher)[0] : plannedTime.match(this.timeMatcher)[0];
+  }
+
+  private static getPauseStartTime(hearingActuals: HearingActualsMainModel): string {
+    const actualTime = hearingActuals.hearingActuals.actualHearingDays[0].pauseDateTimes[0];
+    return actualTime && actualTime.pauseStartTime ? actualTime.pauseStartTime.match(this.timeMatcher)[0] : null;
+  }
+
+  private static getPauseEndTime(hearingActuals: HearingActualsMainModel): string {
+    const actualTime = hearingActuals.hearingActuals.actualHearingDays[0].pauseDateTimes[0];
+    return actualTime && actualTime.pauseEndTime ? actualTime.pauseEndTime.match(this.timeMatcher)[0] : null;
+  }
+
+  private static getEndTime(hearingActuals: HearingActualsMainModel): string {
+    const plannedTime = hearingActuals.hearingPlanned.plannedHearingDays[0].plannedStartTime;
+    const actualTime = hearingActuals.hearingActuals.actualHearingDays[0].hearingEndTime;
+    return actualTime ? actualTime.match(this.timeMatcher)[0] : plannedTime.match(this.timeMatcher)[0];
+  }
+
+  private static replaceTime(time: string, value: string) {
+    if (!time) {
+      return time;
+    }
+    return time.replace(this.timeMatcher, value);
+  }
+
+  public ngOnInit() {
+    this.sub = this.hearingStore.select(fromHearingStore.getHearingActuals)
+      .pipe(
+        filter((state: HearingActualsStateData) => !!state.hearingActualsMainModel)
+      )
+      .subscribe((state: HearingActualsStateData) => {
+        this.hearingActuals = state.hearingActualsMainModel;
+        this.form = this.createForm(this.hearingActuals);
+      });
+  }
+
+  public ngOnDestroy() {
+    this.sub.unsubscribe();
+  }
+
+  public submit(value: any, valid: boolean) {
+    if (valid) {
+      const actuals = {
+        ...this.hearingActuals.hearingActuals,
+        actualHearingDays: [
+          {
+            ...this.hearingActuals.hearingActuals.actualHearingDays[0],
+            hearingStartTime: HearingActualsTimingComponent.replaceTime(this.hearingActuals.hearingActuals.actualHearingDays[0].hearingStartTime, value.hearingStartTime),
+            hearingEndTime: HearingActualsTimingComponent.replaceTime(this.hearingActuals.hearingActuals.actualHearingDays[0].hearingEndTime, value.hearingEndTime),
+            pauseDateTimes: [
+              {
+                pauseStartTime: HearingActualsTimingComponent.replaceTime(this.hearingActuals.hearingActuals.actualHearingDays[0].hearingStartTime, value.pauseStartTime),
+                pauseEndTime: HearingActualsTimingComponent.replaceTime(this.hearingActuals.hearingActuals.actualHearingDays[0].hearingStartTime, value.pauseEndTime)
+              }
+            ]
+          }
+        ]
+      };
+      this.hearingsService.updateHearingActuals('1', actuals).subscribe(() => {
```

Comment:
- hard code hearing ID?
- could we use dispatch an action and effect to fire the API call instead of subscribe it, and catch error in the effect
-  this subscribe doesn't catch error